test/index.test.js

```diff
@@ -1222,4 +1222,46 @@ Previous tag: ''
       expect.assertions(1)
     })
   })
+  describe('config error handling', () => {
+    let output
+    beforeEach(() => {
+      output = []
+      probot.logger.addStream({
+        level: 'trace',
+        stream: { write: log => output.push(log) },
+        type: 'raw'
+      })
+    })
+    it('schema error', async () => {
+      getConfigMock('config-with-schema-error.yml')
+
+      const payload = require('./fixtures/push')
+
+      await probot.receive({
+        name: 'push',
+        payload
+      })
+      expect(output.length).toBe(3)
+      expect(output[1].msg).toMatch('Bad replacer regex')
```

Comment:
Seems like we need to return earlier inside config.js I just hate added try and catch and early returns 😭